docs/main/05-reference/06-workflows.md

```diff
@@ -83,67 +85,110 @@ Refer to the [Kubernetes Pod Spec
 documentation](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
 for more information on the fields above.
 
-To access secrets in the pipeline, refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/).
+### Secrets
+
+To access Secrets in the Pipeline, you can pass in a reference to the Pipeline container's
+`env` using `valueFrom.secretKeyRef` in the standard Kubernetes way.
+
+```yaml
+apiVersion: platform.kratix.io/v1alpha1
+kind: Promise
+metadata:
+  ...
+spec:
+  api:
+    ...
+  workflows:
+    resource:
+      configure:
+        - apiVersion: platform.kratix.io/v1alpha1
+          kind: Pipeline
+          metadata:
+            ...
+          spec:
+            containers:
+              - name: ...
+                image: ...
+                env:
+                  - name: # Env var to store Secret value
+                    valueFrom:
+                      secretKeyRef:
+                        name: # Secret name
+                        key: # Secret key
+```
 
 :::note
 
-Not all fields from the Pod spec are supported. We will add support for more
-fields in the future.
+The Secret must be accessible within the Pipeline's namespace.
 
```

Comment:
any reason to remove this?

Reply:
Added it back with hindsight! It's useful to keep underneath the reference to the k8s Pod documentation 👍